lib/cosmos/dart/app/models/decom_table.rb

```diff
@@ -0,0 +1,2 @@
+class DecomTable < ApplicationRecord
```

Comment:
@ryanatball Should this be deleted? I don't see it used anywhere.

Reply:
yes